client/packages/system/src/Item/Components/VaccineItemSelect/VaccineItemSelect.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {
+  Checkbox,
+  AutocompleteOptionRenderer,
+  FnUtils,
+  AutocompleteMulti,
+  VaccineCourseItemNode,
+} from '@openmsupply-client/common';
+import { useVaccineItems } from 'packages/system/src/IndicatorsDemographics/api/hooks/document/useVaccineItems';
+import { DraftVaccineCourse } from 'packages/system/src/Immunisation/api/hooks/useVaccineCourse';
+
+interface VaccineItemSelectProps {
+  extraFilter?: (item: VaccineCourseItemNode) => boolean;
+  onChange: (newData: Partial<DraftVaccineCourse>) => void;
+  draft: DraftVaccineCourse;
+}
+
+const renderOption: AutocompleteOptionRenderer<VaccineCourseItemNode> = (
+  props,
+  option,
+  { selected }
+): JSX.Element => {
+  return (
+    <li {...props}>
+      <Checkbox checked={selected} />
+      <span
+        style={{
+          fontWeight: 700,
+          whiteSpace: 'nowrap',
+          width: 100,
+        }}
+      >
+        {option.item.name ?? ''}
+      </span>
+    </li>
+  );
+};
+
+export const VaccineItemSelect = ({
+  // extraFilter,
+  onChange,
+  draft,
+}: VaccineItemSelectProps) => {
+  const { data } = useVaccineItems();
+
+  const onChangeSelectedItems = (selectedItems: VaccineCourseItemNode[]) => {
+    onChange({ vaccineCourseItems: selectedItems });
+  };
+
+  const options =
+    data?.nodes?.map(item => {
+      return {
+        id: FnUtils.generateUUID(),
+        item: {
+          id: item.id,
+          name: item.name,
+        },
+      } as VaccineCourseItemNode;
+    }) ?? ([] as VaccineCourseItemNode[]);
+
+  return (
+    <AutocompleteMulti
+      isOptionEqualToValue={(option, value) => option.item.id === value.item.id}
+      getOptionLabel={option => `${option.item.name}`}
+      defaultValue={
```

Comment:
i think this can be value instead of default value - that will resolve the console error when you set a vaccine item :)